org.jitsi.jitsi-meet.yaml

```diff
@@ -0,0 +1,60 @@
+---
+app-id: org.jitsi.jitsi-meet
+base: org.electronjs.Electron2.BaseApp
+base-version: '19.08'
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: jitsi-meet-run
+separate-locales: false
+tags: []
+finish-args:
+  - --device=all
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=x11
+  - --talk-name=org.freedesktop.Notifications
+modules:
+  - name: unappimage
+    buildsystem: simple
+    build-commands:
+      - make -C squashfs-tools -j ${FLATPAK_BUILDER_N_JOBS} install INSTALL_DIR=/app/bin
+    sources:
+      - type: git
+        url: https://github.com/refi64/unappimage
+        commit: d7f86f2a0d7ec3a69211125207d5f127386b849a
+
+  - name: jitsi-meet
+    buildsystem: simple
+    build-commands:
+      - unappimage jitsi-meet-x86_64.AppImage
+      - rm jitsi-meet-x86_64.AppImage
+
+      - sed -e 's/Exec=.*/Exec=jitsi-meet-run/'
+            -e 's/Icon=.*/Icon=org.jitsi.jitsi-meet/'
+              squashfs-root/jitsi-meet.desktop > org.jitsi.jitsi-meet.desktop
+      - cp squashfs-root/jitsi-meet.png org.jitsi.jitsi-meet.png
+
+      - install -D jitsi-meet-run -t /app/bin
+      - install -Dm644 org.jitsi.jitsi-meet.appdata.xml -t /app/share/metainfo
+      - install -Dm644 org.jitsi.jitsi-meet.desktop -t /app/share/applications
+      - install -Dm644 org.jitsi.jitsi-meet.png -t /app/share/icons/hicolor/512x512/apps
+
+      - mv squashfs-root /app/bin/jitsi-meet
+    sources:
+      - type: script
+        dest-filename: jitsi-meet-run
+        commands:
+          - zypak-wrapper /app/bin/jitsi-meet/jitsi-meet "$@"
+      - type: file
+        path: org.jitsi.jitsi-meet.appdata.xml
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html